ci/jenkins/Build.groovy.j2

```diff
@@ -21,7 +21,7 @@ def fsim_test(image) {
 
 def cmake_build(image, path, make_flag) {
   sh (
-    script: "${docker_run} --env CI_NUM_EXECUTORS ${image} ./tests/scripts/task_build.py --sccache-bucket tvm-sccache-prod",
+    script: "${docker_run} --env CI_NUM_EXECUTORS ${image} ./tests/scripts/task_build.py --sccache-bucket tvm-sccache-prod --build-dir ${path}",
```

Comment:
Actually this seems more broken than I initially thought, what do you think about just fixing the build with the `rocm_device_api.cc` change and reverting the `Build.groovy.j2` changes? https://github.com/apache/tvm/pull/13335/files#r1023416998 will fix up the infra to make things work then we can iterate on actually fixing the full build + test

Reply:
No problem~ thank you very much for your help!